variables.md

```diff
@@ -113,6 +118,62 @@ PERF_KERNEL | boolean | false | Enables kernel performance testing.
 PERF_INSTALL | boolean | false | Enables kernel performance testing installation part.
 PERF_SETUP | boolean | false | Enables kernel performance testing deployment part.
 PERF_RUNCASE | boolean | false | Enables kernel performance testing run case part.
+_SECRET_PUBLIC_CLOUD_REST_URL | string | "https://publiccloud.qa.suse.de/vault" | Vault server URL
```

Comment:
I think this should be dropped . in my understanding this file is about variables which can be used in **tests** while this is variables should be defined in the worker by design . @cfconrad ?

Reply:
Yes, the idea is to specify such variable only in the worker, as it will also not be visible in the `vars.json` file, because of the `_SECRET` prefix.

Reply:
But why you don't wanna document this variable? It is possible to specify the variable as all others, so no need to hide it in the docs.

Reply:
because in my understanding `variables.md` suppose to describe variables which influence a test flow and this are more "setup worker" variables

Reply:
But it is used in test. In `publiccloud::provider`.